qcore/constants.py

```diff
@@ -147,7 +147,7 @@ class ProcessType(ExtendedStrEnum):
         False,
         False,
         "time python $IMPATH/calculate_ims.py {sim_dir}/BB/Acc/BB.bin b -o {sim_dir}/IM_calc/ -np {np} -i "
-        "{sim_name} -r {fault_name} -c {component} -t s {extended} {simple}",
+        "{sim_name} -r {fault_name} -t s {component} {extended} {simple} {advanced_IM}",
```

Comment:
Wouldn't this run it with the advanced im flags?

Reply:
the default value of the advanced_IM is empty.
so one of {component} and {advanced_IM} will be empty while the other have value.